plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/block-settings/index.tsx

```diff
@@ -13,26 +18,27 @@ export const ProductGalleryThumbnailsBlockSettings = ( {
 	attributes,
 	setAttributes,
 }: ProductGalleryThumbnailsSettingsProps ) => {
-	const maxNumberOfThumbnails = 8;
-	const minNumberOfThumbnails = 3;
-	const { numberOfThumbnails } = attributes;
+	const { thumbnailSize } = attributes;
 
 	return (
-		<RangeControl
-			label={ __( 'Number of Thumbnails', 'woocommerce' ) }
-			value={ numberOfThumbnails }
-			onChange={ ( value: number ) =>
+		<UnitControl
+			label={ __( 'Thumbnail Size', 'woocommerce' ) }
+			value={ thumbnailSize }
+			onChange={ ( value: string | undefined ) =>
 				setAttributes( {
-					numberOfThumbnails: Math.round( value ),
+					thumbnailSize: value || '20%',
```

Comment:
Should this be consistent with the [default value](https://github.com/woocommerce/woocommerce/pull/56069/files#diff-c619be550dcc226ec082db06e8f7ec961c053521c64a211ddb4e3a90ab5df833R16)?
```suggestion
					thumbnailSize: value || '33%',
```